components/CreateWorkout/CreateWorkout.jsx

```diff
@@ -123,30 +123,30 @@ export default ({ navigation }) => {
         } else if (itemState.length === 0) {
           alert('Please add at least one exercise');
         } else {
-          let muscleGroups = itemState[0].muscleGroups; 
+          let { muscleGroups } = itemState[0];
           if (itemState.length >= 2) { // for showing top 2 muscle groups
-            muscleGroups += " ";
+            muscleGroups += ' ';
             muscleGroups += itemState[1].muscleGroups;
           }
           const newWorkout = {
             name,
             lastPerformed: 'n/a',
             id: nextWorkoutId,
-            muscleGroups: muscleGroups,
+            muscleGroups,
             color: itemState[0].color,
```

Comment:
If kept, the color would be stored in the database which defeats the purpose of rendering it on the front end. (Is this needed for front end rendering or redux or something?) If not, then for this task, I think you should also get rid of all instances relating color to redux and db.

Reply:
I removed those instances when I fixed the cycle bug, they should all be gone now